pkgs/native_assets_builder/lib/src/build_runner/build_runner.dart

```diff
@@ -659,29 +684,17 @@ ${e.message}
     final depFile = File.fromUri(
       outputDirectory.resolve('../hook.dill.d'),
     );
+    final dependenciesHashFile = File.fromUri(
+      outputDirectory.resolve('../hook.dependencies_hash_file.json'),
```

Comment:
Will we re-build the kernel file if the Dart SDK has changed? (otherwise invoking the old kernel on newer SDK may just crash)

Reply:
This is why you review the code! 🤓 🙏